.github/workflows/chromatic.yml

```diff
@@ -4,68 +4,34 @@ on:
   schedule:
     - cron: '0 0 * * *'
   workflow_dispatch:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - 'packages/frontend/@n8n/design-system/**'
+      - '.github/workflows/storybook.yml'
   pull_request_review:
     types: [submitted]
 
 concurrency:
-  group: chromatic-${{ github.event.pull_request.number || github.ref }}-${{github.event.review.state}}
+  group: chromatic-${{ github.event.pull_request.number || github.ref }}
   cancel-in-progress: true
 
 jobs:
-  get-metadata:
-    name: Get Metadata
-    runs-on: ubuntu-latest
-    if: github.event.review.state == 'approved'
-    steps:
-      - name: Check out current commit
-        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
-        with:
-          ref: ${{ github.event.pull_request.head.sha }}
-          fetch-depth: 2
-
-      - name: Determine changed files
-        uses: tomi/paths-filter-action@32c62f5ca100c1110406e3477d5b3ecef4666fec # v3.0.2
-        id: changed
-        if: github.event_name == 'pull_request_review'
-        with:
-          filters: |
-            design_system:
-              - packages/frontend/@n8n/design-system/**
-              - .github/workflows/chromatic.yml
-
-    outputs:
-      design_system_files_changed: ${{ steps.changed.outputs.design_system == 'true' }}
-      is_community_pr: ${{ contains(github.event.pull_request.labels.*.name, 'community') }}
-      is_pr_target_master: ${{ github.event.pull_request.base.ref == 'master' }}
-      is_dispatch: ${{ github.event_name == 'workflow_dispatch' }}
-      is_pr_approved: ${{ github.event.review.state == 'approved' }}
-
   chromatic:
-    needs: [get-metadata]
     if: |
-      needs.get-metadata.outputs.is_dispatch == 'true' ||
-      (
-       needs.get-metadata.outputs.design_system_files_changed == 'true' &&
-       needs.get-metadata.outputs.is_community_pr == 'false' &&
-       needs.get-metadata.outputs.is_pr_target_master == 'true' &&
-       needs.get-metadata.outputs.is_pr_approved == 'true'
-      )
-    runs-on: ubuntu-latest
+      github.event.review.state == 'approved' &&
+      !contains(github.event.pull_request.labels.*.name, 'community')
+    runs-on: blacksmith-2vcpu-ubuntu-2204
     steps:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
         with:
           fetch-depth: 0
```

Comment:
You could remove this fetch-depth argument. 0 pulls everything in the commit history, leaving it empty is the default to just pull this commit.